pokemon.js

```diff
@@ -0,0 +1,117 @@
+document.addEventListener("DOMContentLoaded", () =>{
+    // Global selectors
+    let getPokemon = document.querySelector("#getPokemon")
+    let battle = document.querySelector("#battle")
+    let battlelist = document.querySelector("#battlelist")
+    let rightImage = document.querySelector("#rightImage")
+    let leftImage = document.querySelector("#leftImage")
+    let audio = document.querySelector("#audio")
+    let pokemon = []
+    let res
+    // first pokemon is unique
+    const randomPokemon1 = async () =>{
```

Comment:
You can try to make this function more flexible by adding the div you want to append to as a variable